### PR TITLE
network: update names in generated certs

### PR DESCRIPTION
**Pull request by thc202**
Update the names of the root CA cert and issued server certificates, rename also related code constants.
Update the help to reflect the changes.

### CODE REVIEW

**Comment by thc202** [comment]
I didn't rename the alias nor the passphrase of the KeyStore as those are internal, happy to.

**Review by kingthorin — APPROVED** [review]
LGTM

File: addOns/network/src/main/javahelp/help/contents/options/servercertificates.html [hunk 1]
```diff
@@ -92,9 +92,9 @@ <h3>Generate</h3>
 	<h3>Import</h3>
 	<p>
 		When you're using multiple ZAP installation and you want to use the same
-		Root CA certificate, so you can import it. Simply use one installation of OWASP ZAP
+		Root CA certificate, so you can import it. Simply use one installation of ZAP
 		to generate one Root CA certificate.<br>
-		Copy the file 'OWASP ZAP/config.xml' from your users home directory to
+		Copy the file 'config.xml' from the ZAP's home directory to
```
Comment (ricekot) [inline addOns/network/src/main/javahelp/help/contents/options/servercertificates.html:97]:
```suggestion
		Copy the file 'config.xml' from ZAP's home directory to
```

Comment (thc202) [reply addOns/network/src/main/javahelp/help/contents/options/servercertificates.html]: Done.

**Review by kingthorin — APPROVED** [review]

**Review by ricekot — APPROVED** [review]
Thank you!

**Pull request merged**